src/main/java/io/vertx/gsoc2023/qotd/QuoteOfTheDayVerticle.java

```diff
@@ -55,22 +60,43 @@ private void getHandler(PgPool pool, RoutingContext ctx) {
 		}).onSuccess(x -> response.end(Json.encodePrettily(receivedQuotes)))
 				.onFailure(v -> response.end(v.getCause().getLocalizedMessage()));
 	}
+	
+	private void postHandler(PgPool pool, RoutingContext ctx) {
+		Quotes quote = ctx.body().asPojo(Quotes.class);
+		HttpServerResponse response = ctx.response();
+		response.setChunked(true);
+		
+		if(quote.getQuote().toString().equals("") || quote.getQuote().toString()==null) {
+			response.setStatusCode(400);
+			response.end("Please provide a valid response");
+			return;
+		}
+		
+		String quoteString = quote.getQuote().toString();
+		String author = quote.getAuthor().toString() == null || quote.getAuthor().toString().equals("") ? "Unknown": quote.getAuthor().toString();
+		
+		pool.getConnection().compose(conn -> {
+			return conn.query("INSERT INTO quotes (text, author) VALUES ('" + quoteString + "', '" + author + "')")
+					.execute()
+					.eventually(v -> conn.close())
+					.onFailure(fail -> response.end(fail.getCause().getLocalizedMessage()))
+					.onSuccess(ar -> response.end("Quote saved successfully"));
+		});
+	}
 
-  private PgPool setupPool(JsonObject config) {
-    PgConnectOptions connectOptions = new PgConnectOptions()
-      .setUser(config.getString("dbUser", "quotes"))
-      .setPassword(config.getString("dbPassword", "super$ecret"))
-      .setDatabase(config.getString("dbName", "quotes"))
-      .setHost(config.getString("dbHost", "localhost"))
-      .setPort(config.getInteger("dbPort", 5432));
-    return PgPool.pool(vertx, connectOptions, new PoolOptions());
-  }
+	private PgPool setupPool(JsonObject config) {
+		PgConnectOptions connectOptions = new PgConnectOptions().setUser(config.getString("dbUser", "quotes"))
+				.setPassword(config.getString("dbPassword", "super$ecret"))
+				.setDatabase(config.getString("dbName", "quotes")).setHost(config.getString("dbHost", "localhost"))
+				.setPort(config.getInteger("dbPort", 5432));
+		return PgPool.pool(vertx, connectOptions, new PoolOptions());
+	}
 
-  private Router setupRouter() {
-    return Router.router(vertx);
-  }
+	private Router setupRouter() {
+		return Router.router(vertx);
```

Comment:
Please make sure to configure your IDE so that formatting changes are not committed.